java/test/src/main/java/io/ray/test/ParallelActorTest.java

```diff
@@ -0,0 +1,140 @@
+package io.ray.test;
+
+import com.google.common.base.Preconditions;
+import io.ray.api.ActorHandle;
+import io.ray.api.ObjectRef;
+import io.ray.api.Ray;
+import io.ray.api.parallelactor.*;
+import io.ray.api.parallelactor.strategy.RandomStrategy;
+import io.ray.api.parallelactor.strategy.RoundRobinStrategy;
+import io.ray.runtime.exception.RayActorException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+@Test(groups = "cluster")
+public class ParallelActorTest extends BaseTest {
+
+  private static final Logger LOG = LoggerFactory.getLogger(ParallelActorTest.class);
+
+  private static class A {
+    private int value = 0;
+
+    public int incr(int delta) {
+      value += delta;
+      return value;
+    }
+
+    public int getValue() {
+      return value;
+    }
+
+    public int add(int a, int b) {
+      return a + b;
+    }
+
+    public int getThreadId() {
+      return 1000000 + (int) Thread.currentThread().getId();
+    }
+  }
+
+  /// 1. How to support concurrency group on parallel actor?
+  /// 2. how to align all functionalities of ActorCaller/ActorTaskCaller, we need to extends from
+  // ActorHandle?
+  /// 3. How do we reuse APIs ActorCreator for parallelActorCreator, like setRuntimeEnv
+  /// 4. The router is in the caller, to decide which index we should route. How to solve? Move to
+  // callee?
+  /// 5. We used function manager.
+  public void testRoundRobinStrategy() {
+    ParallelActor<A> actor =
+        Parallel.actor(A::new).setStrategy(new RoundRobinStrategy(10)).remote();
+
+    /// TODO: Why we set the delta so large? because it's cast to short otherwise.
+    ObjectRef<Integer> obj0 = actor.task(A::incr, 1000000).remote(); // in instance 0
+    ObjectRef<Integer> obj1 = actor.task(A::incr, 2000000).remote(); // in instance 1
+    ObjectRef<Integer> obj2 = actor.task(A::incr, 3000000).remote(); // in instance 2
```

Comment:
Can we add a getPid method to the actor class and verify that all of them are in the same worker process?